core/src/main/scala/org/apache/spark/sql/execution/CoprocessorRDD.scala

```diff
@@ -220,13 +153,13 @@ case class RegionTaskExec(child: SparkPlan,
     numIndexRangesScanned: SQLMetric,
     numDowngradeRangesScanned: SQLMetric,
     downgradeDagRequest: TiDAGRequest
-  ): (Int, Iterator[InternalRow]) => Iterator[UnsafeRow] = { (index, iter) =>
+  ): (Int, Iterator[InternalRow]) => Iterator[InternalRow] = { (_, iter) =>
```

Comment:
since we are not using the index anymore, can we simply delete it?

Reply:
This can be done in future PR.